util/ci_utils.sh

```diff
@@ -178,7 +178,7 @@ build_all() {
     echo
 }
 
-build_wheel() {
+build_pip_conda_package() {
```

Comment:
For local testing, we'd like the option to only build one of the formats (eg if conda environment is not installed):
`build_pip_conda_package conda|pip|both` with `both` as default

Reply:
done.